src/main/webapp/app/components/sidebar/NavigationSidebar.tsx

```diff
@@ -289,9 +290,9 @@ const mapStoreToProps = ({ layoutStore, authStore }: IRootStore) => ({
   isNavSidebarCollapsed: layoutStore.isNavigationSidebarCollapsed,
   navigationSidebarWidth: layoutStore.navigationSidebarWidth,
   isAuthenticated: authStore.isAuthenticated,
-  isAdmin: hasAnyAuthority(authStore.account.authorities, [AUTHORITIES.ADMIN]),
-  isCurator: hasAnyAuthority(authStore.account.authorities, [AUTHORITIES.CURATOR]),
-  isUser: hasAnyAuthority(authStore.account.authorities, [AUTHORITIES.USER]),
+  isAdmin: hasAnyAuthority(authStore.account.authorities ?? [], [AUTHORITIES.ADMIN]),
+  isCurator: hasAnyAuthority(authStore.account.authorities ?? [], [AUTHORITIES.CURATOR]),
+  isUser: hasAnyAuthority(authStore.account.authorities ?? [], [AUTHORITIES.USER]),
   account: authStore.account,
```

Comment:
See my other reply comment to Calvin. Once we fix the backend models. These checks won't be necessary.